misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/BTestRunner.java

```diff
@@ -277,9 +278,13 @@ private void execute(TestSuite suite, ClassLoader testClassLoader) {
     private void executeBeforeSuiteFunctions(TestSuite suite, ClassLoader classLoader, Scheduler scheduler,
                                              AtomicBoolean shouldSkip, AtomicBoolean shouldSkipAfterSuite) {
         suite.getBeforeSuiteFunctionNames().forEach(test -> {
+            Object value = null;
```

Comment:
Let's move variable declaration for 'value' to line 284 as well as it's not used outside the try block.

Reply:
Moved all value declarations inside the try blocks